mk/uvmt/xrun.mk

```diff
@@ -206,6 +206,9 @@ XRUN_RUN_FLAGS  += -nowarn RNDNOXCEL
 # Probes
 XRUN_RUN_FLAGS  += -nowarn PRLDYN
 
+# Physical repository related to logical library in a cds.lib does not exist
+XRUN_COMP_FLAGS += -nowarn DLCPTH
```

Comment:
I don't know the causes or consequences of these warnings. Are we sure we want to have the warnings disabled by core-v-verif? I feel a bit uneasy about disabling a warning that I don't understand.

Reply:
Leaving this flag out causes _a lot_ of warnings about missing paths to physical technology-libraries, which are not used for rtl-level verification in core-v-verif, so I propose that we exclude this warning to have compile logs that are humanly readable without too much effort.

Reply:
Then I understand. Agreed, let's suppress this warning.